onnxruntime/core/session/inference_session.h

```diff
@@ -543,6 +542,11 @@ class InferenceSession {
   std::vector<std::string> transformers_to_enable_;
 #endif
 
+#if !defined(ORT_MINIMAL_BUILD) || defined(ORT_EXTENDED_MINIMAL_BUILD)
+  Status PartitionOrtFormatModel(onnxruntime::Graph& graph, const ExecutionProviders& providers,
```

Comment:
Noticed that some functions use ORT_MUST_USE_RESULT. We're not consistent. Should new functions have this as well?

Reply:
Not sure what the policy should be. I thought we added that to the public APIs initially, and following that it seemed to bleed down a bit. I thought the VS Code Analysis could find issues where a return value is ignored, so it may not be necessary to add ORT_MUST_USE_RESULT to internal code.